chess-service/src/main/java/com/javamonkeys/api/GameService.java

```diff
@@ -3,56 +3,218 @@
 import com.javamonkeys.dao.game.Game;
 import com.javamonkeys.dao.game.GameDao;
 import com.javamonkeys.dao.game.GameNotFoundException;
-import com.javamonkeys.dao.user.User;
+import com.javamonkeys.dao.game.IGameDao;
+import com.javamonkeys.dao.user.*;
+
+import org.hibernate.Query;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
-import org.springframework.web.bind.annotation.RestController;
-import org.springframework.web.bind.annotation.RequestParam;
-
-/**
- * Created by ose on 4/3/15.
- */
+import org.springframework.http.ResponseEntity;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.*;
+
+import javax.inject.Inject;
+
+
 @RestController
 @RequestMapping("/game")
-public class GameService {
+public class GameService implements IGameService {
 
-//    @RequestMapping(produces = MediaType.APPLICATION_JSON_VALUE)
-    @RequestMapping(value = "/get-game", produces = MediaType.APPLICATION_JSON_VALUE)
-    public @ResponseBody Game getGame() {
-        return new Game(new User("email","no password"));
+    @Inject
+    private GameDao gameDao;
+
+    @Inject
+    private UserDao userDao;
+
+    @Inject
+    private SessionFactory hibernateSessionFactory;
+
+    //    @Inject
+    //    RequestInfo requestInfo;
+    static class requestInfo {
+        public static final int userId = 1;
     }
```

Comment:
Why do you need this class? Note: classes names must start with letter in uppercase